modules/openapi-generator/src/main/resources/python-nextgen/api_response.mustache

```diff
@@ -0,0 +1,23 @@
+from __future__ import annotations
+from typing import Any, List, Optional
+from pydantic import StrictInt, StrictStr
+
+class ApiResponse:
+    """
+    API response object
+    """
+
+    status_code: Optional[StrictInt]
+    headers: Optional[Dict[StrictStr, StrictStr]]
+    data: Optional[Any]
+    raw_data: Optional[Any]
+
+    def __init__(self,
+                 status_code=None,
+                 headers=None,
+                 data=None,
```

Comment:
What do this and raw_data store? Json? How about adding a more descriptive type hint or using generics?

Reply:
updated with description